orc8r/cloud/test/scale/basic_test.go

```diff
@@ -0,0 +1,429 @@
+	ClientPFXFilepath = "/Users/hcgatewood/Desktop/tmp/orc8r_deployment/secrets/admin_operator.pfx"
+	ClientPFXPassword = ""
+)
+
+// var CloudURL = "localhost:9443"
+var CloudURL = CloudCertURL
+
+var (
+	shouldRegenClusterInfo = true
+	shouldClean            = true
+	shouldPortForward      = true
+	shouldCreate           = true
+	shouldResetGateways    = true
+)
+
+func TestBasicScale(t *testing.T) {
+	glog.Info("Get cluster info")
+	cluster, err := testlib.GetClusterFromFile()
+	assert.NoError(t, err)
+	if shouldRegenClusterInfo {
+		cluster, err = testlib.GetClusterInfo()
+		assert.NoError(t, err)
+		err = testlib.SetClusterInfo(cluster)
+		assert.NoError(t, err)
+	}
+
+	glog.Info("Read server cert")
+	rootCAs, err := x509.SystemCertPool()
+	require.NoError(t, err)
+	pemBytes, err := ioutil.ReadFile(RootCertFilepath)
+	require.NoError(t, err)
+	ok := rootCAs.AppendCertsFromPEM(pemBytes)
+	require.True(t, ok)
+
+	glog.Info("Read client cert")
+	pfxBytes, err := ioutil.ReadFile(ClientPFXFilepath)
+	require.NoError(t, err)
+	key, cert, err := pkcs12.Decode(pfxBytes, ClientPFXPassword)
+	require.NoError(t, err)
+
+	glog.Info("Get REST API client")
+	tlsConfig := &tls.Config{
+		RootCAs:      rootCAs,
+		ServerName:   fmt.Sprintf("*.%s", CloudCertURL),
+		Certificates: []tls.Certificate{{Certificate: [][]byte{cert.Raw}, PrivateKey: key}},
+	}
+	tlsConfig.BuildNameToCertificate()
+	httpClient := &http.Client{Transport: &http.Transport{TLSClientConfig: tlsConfig}}
+	openAPIClient := oclient.NewWithClient(fmt.Sprintf("api.%s", CloudURL), client.DefaultBasePath, []string{"https"}, httpClient)
+	c := client.New(openAPIClient, nil)
+
+	glog.Info("Get K8s client")
+	kconfigBytes, err := ioutil.ReadFile(KconfigPath)
+	require.NoError(t, err)
+	kRestConfig, err := clientcmd.RESTConfigFromKubeConfig(kconfigBytes)
+	require.NoError(t, err)
+	k, err := kubernetes.NewForConfig(kRestConfig)
+	require.NoError(t, err)
+
+	promClient, err := api.NewClient(api.Config{Address: "http://localhost:9090"})
+	require.NoError(t, err)
+	p := prom.NewAPI(promClient)
+
+	glog.Info("Get DB connection string")
+	connStrSecret, err := k.CoreV1().Secrets("orc8r").Get("orc8r-controller", metav1.GetOptions{})
+	require.NoError(t, err)
+	connStr := string(connStrSecret.Data["postgres.connstr"])
+	require.NotEmpty(t, connStr)
+
+	if shouldClean {
+		glog.Info("Clear DB")
+		pod := getOrchestratorPod(t, k)
+		postgresClearCmd := fmt.Sprintf(PostgresClearCmdTemplate, connStr)
+		kubectlExec(t, k, kRestConfig, pod, postgresClearCmd)
+
+		glog.Info("Bounce application pods")
+		err = k.CoreV1().Pods("orc8r").DeleteCollection(nil, metav1.ListOptions{LabelSelector: "app.kubernetes.io/name=orc8r"})
+		require.NoError(t, err)
+		err = k.CoreV1().Pods("orc8r").DeleteCollection(nil, metav1.ListOptions{LabelSelector: "app.kubernetes.io/name=lte-orc8r"})
+		require.NoError(t, err)
+
+		glog.Info("Wait for pods to come up")
+		time.Sleep(1 * time.Minute)
```

Comment:
Could this lead to test instability? I'd prefer this changed if there's a way to check if the pods are up.

Reply:
Agreed this is non-ideal. Am deferring this fix as part of the stabilization effort